RxJavaTemplate/buildSrc/src/main/java/Versions.kt

```diff
@@ -1,60 +1,60 @@
 object Versions {
-     const val BUILD_GRADLE_VERSION = "4.2.2"
```

Comment:
@lydiasama Is it an unrelated change 😄  ?
Same for others.

Reply:
It's from my AS formatting automatically. 😱

Reply:
@hoangnguyen92dn why did we use 5 spaces to tab in your last PR?

Reply:
I think my format is correct, we use 4 spaces instead of 5. @doannimble

Reply:
@lydiasama Yeah, I know your format is correct and comes from AS.
As I used to be warned it when pushing code with auto-format code from 7-11 project, then I just wanna make a confirmation about this one :D.

Reply:
@doannimble Ok. Then I'll generate another ticket to format this file. It's just a format imo. 😌

Reply:
@lydiasama Totaly agree with you, it does not a matter or a big thing. But it would be great if we can make our process consistent even it looks a bit cumbersome 🙏